plugins/askrene/askrene.c

```diff
@@ -893,14 +904,15 @@ static struct command_result *json_askrene_listreservations(struct command *cmd,
 	struct askrene *askrene = get_askrene(cmd->plugin);
 	struct json_stream *response;
 
+	/* FIXME: We could allow layer names here? */
```

Comment:
Yes, why not?
The only question is: if I specify a list of layers here, would I want to get a list of reservations
in which some of them are global or not?

Reply:
Yes, indeed.  One reason I didn't implement it, since I can't tell how it would be used.